nimsuggest/nimsuggest.nim

```diff
@@ -730,16 +730,17 @@ proc handleCmdLine(cache: IdentCache; conf: ConfigRef) =
   graph.onMarkUsed = proc (g: ModuleGraph; info: TLineInfo; s: PSym; usageSym: var PSym; isDecl: bool) =
     suggestSym(g, info, s, usageSym, isDecl)
   graph.onSymImport = graph.onMarkUsed # same callback
-  if self.loadConfigsAndProcessCmdLine(cache, conf, graph):
+  if self.loadConfigsAndProcessCmdLine(cache, conf, graph, argv):
     # defaulting to customizing for the C backend matches what
     # 'nim check' does
     customizeForBackend(graph, conf, backendC)
     mainCommand(graph)
 
 when isMainModule:
+  let argv = getExecArgs()
```

Comment:
It can stay like this, but in the future, please pass the result to the parameter directly where possible. That is: `handleCmdLine(..., ..., getExecArgs())`.